.github/workflows/builds.yml

```diff
@@ -2489,12 +2496,20 @@ jobs:
                 linux-headers-$(uname -r)
 
       - name: Checkout Netmap repository
+        if: steps.netmap-cache.outputs.cache-hit != 'true'
```

Comment:
The docs suggest a syntax like:
```
if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
```
however, it seems to work as-is.

Reply:
this is straight from some of the docs about caching